exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,76 @@
+{
+   "#": [
+     "Not sure what to type here, come back to this later."
+   ],
+   "cases": [
+      {
+         "description": "TestSingleBookPrice",
```

Comment:
In retrospect this probably isn't needed.  I have a property that just returns the price of a single book.  The test is just confirming that it equals 8.